fhir-persistence/src/test/java/com/ibm/fhir/persistence/test/common/AbstractIncludeRevincludeTest.java

```diff
@@ -518,10 +518,10 @@ private void checkIncludeAndRevIncludeResources(List<Resource> resources, int nu
         assertTrue(numOfMatchedOrgs == foundPatientOrgIds.size());
     }
     /**
-     * This test queries page of Patient with inclusion of the referenced managing organizations. 
+     * This test queries page of Patient with inclusion of the referenced managing organizations.
      * and also requests the reverse inclusion of Observations.
      * It verifies:
-     * (1) the number of patients in the result page; 
+     * (1) the number of patients in the result page;
      * (2) the returned managing organizations and observations are related to the patients of the same page;
      * (3) all related managing organizations are in the same page.
```

Comment:
Javadoc will lose this nice formatting.  Use `<p>` above "It verifies:" and add `<ul>` or `<ol>` and `<li>` tags for the list.

Reply:
ok, I see. will add these html tags